bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -108,11 +109,21 @@ func getConfigFromCache(pathDir string, kfDef *kfdefsv3.KfDef) ([]byte, error) {
 // GetPlatform will return an implementation of kftypesv3.GetPlatform that matches the platform string
 // It looks for statically compiled-in implementations, otherwise throws unrecognized error
 func getPlatform(kfdef *kfdefsv3.KfDef) (kftypesv3.Platform, error) {
+	kfdefBytes, err := yaml.Marshal(kfdef)
```

Comment:
Why is this function taking a kfDef  and not a KfConfig?

Reply:
We should eventually make it take kfconfig after all platforms change to take kfconfig.
But for now, other platforms are taking a kfdef